packages/calcite-components/src/components/radio-button/radio-button.tsx

```diff
@@ -291,11 +304,25 @@ export class RadioButton
     });
   }
 
+  private updateTabIndexOfOtherRadioButtonsInGroup(): void {
+    const radioButtons = this.queryButtons();
+    const otherFocusableRadioButtons = radioButtons.filter(
+      (radioButton) => radioButton.guid !== this.guid && !radioButton.disabled
+    );
+    otherFocusableRadioButtons.forEach((radiobutton) => {
+      forceUpdate(radiobutton);
+    });
+  }
+
   private getTabIndex(): number | undefined {
     if (this.disabled) {
       return undefined;
     }
-    return this.checked || this.isDefaultSelectable() ? 0 : -1;
+    if (this.checked || this.isFocusable()) {
```

Comment:
No love for the previous ternary? 😭

Reply:
Oops, there was some extra code in `if` block earlier. Will revert to ternary.